### PR TITLE
REST-119: Add GitHub Actions setup and usage guide for Angular

### PR DESCRIPTION
**Pull request by ashutosh-bisoyi**
TASK: https://pinestem.com/dashboard.html#/tasks/quick/REST-119/details/?companyId=453&isPrevNext=1

Added a comprehensive document explaining GitHub actions setup and usage guides for angular applications.

### CODE REVIEW

**Comment by osm-vishnukyatannawar** [comment]
Fix the folder name for ci-setups.
Else a new folder will be created again I think.

File: ci-setup/angular-github.md [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+You can modify the above as per your needs. For example if you want to run the jobs for merge requests targeting any release branch you can modify the condition as following:
+
+```yaml
+on:
+  pull_request:
+    branches: [ "main", "development", "release.*" ]
+```
+
+### Writing Jobs for Linting and Building
+Create jobs within each stage to perform linting and building tasks. We will have to jobs in this workflow. One is linting and another is building.
```
Comment (osm-vishnukyatannawar) [inline ci-setup/angular-github.md]: Remove extra space after full stop.

File: ci-setup/angular-github.md [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+In the `lint` and `build` stage, the workflow installs the necessary dependencies using Node.js and npm. It then runs the linting process on the codebase to check for any coding standards violations or errors.
+
+![Linting Job Preview](assets/angular-github_linting-job-preview.png)
+![Linting Job Preview](assets/angular-github_linting-job-start.png)
+
+- If Linting or Build Fails:
+  - The workflow reports issues in the job logs.
+  - The Pull Request status is updated to indicate that the workflow failed.
+  - Developers review the errors in the job logs and make necessary code changes.
+  -
```
Comment (osm-vishnukyatannawar) [inline ci-setup/angular-github.md]: Remove this empty point.

**Comment by ashutosh-bisoyi** [comment]
Yes @osm-vishnukyatannawar I missed to update here. It's done

**Review by osm-vishnukyatannawar — CHANGES_REQUESTED** [review]
Over all looks good.

File: ci-setup/angular-github.md (hunk 2 above)
Comment (ashutosh-bisoyi) [reply ci-setup/angular-github.md]: Removed the empty point

**Review by osm-vishnukyatannawar — APPROVED** [review]

**Pull request merged**